.stitch/designs/chat_desktop.html

```
@@ -0,0 +1,326 @@
+</div>
+</div>
+</nav>
+<!-- Main Content Canvas -->
+<main class="flex flex-1 overflow-hidden">
+<!-- Pane 1: Conversation Sidebar -->
+<aside class="w-80 flex flex-col bg-surface-container-low border-r border-transparent">
+<div class="p-6">
+<h1 class="font-headline text-2xl font-extrabold tracking-tight mb-4">Messages</h1>
+<div class="relative group">
+<span class="material-symbols-outlined absolute left-3 top-1/2 -translate-y-1/2 text-on-surface-variant text-sm">search</span>
+<input class="w-full bg-surface-container-highest border-none rounded-xl pl-10 pr-4 py-3 text-sm focus:ring-2 focus:ring-primary focus:ring-offset-0 transition-all placeholder:text-on-surface-variant/60" placeholder="Search conversations..." type="text"/>
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)